Dockerfile

```diff
@@ -41,6 +41,8 @@ WORKDIR /build-node
 COPY requirements.txt /build-node/requirements.txt
 
 RUN python3 -m venv --system-site-packages env
+# TODO: delete this
```

Comment:
Question. Is this something you need to delete before pushing? Is it only for testing purposes? Is it a temporary solution for something?

Reply:
Yeah, that was for force removing pip packages, I will delete this before merging